restcomm/restcomm.sms/src/main/java/org/restcomm/connect/sms/SmsSession.java

```diff
@@ -135,7 +154,8 @@ private void inbound(final Object message) throws IOException {
                 }
             }
             // Store the last sms event.
-            last = new SmsSessionRequest(from, to, body, customRequestHeaderMap);
+
+            last = new SmsSessionRequest(from, to, body, Encoding.GSM, customRequestHeaderMap, this.tlvSet);
```

Comment:
@gvagenas removed.
I have added extra constructors to `SmsSessionRequest`. Additionally i believe it needs further cleanup